lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -4589,6 +4617,31 @@ StringRef ModuleEmitter::getNameRemotely(Value value,
       xmrString.append(xmr.getTerminal());
       return StringAttr::get(value.getContext(), xmrString);
     }
+
+    if (auto xmrRef = dyn_cast<XMRRefOp>(wireInput)) {
+      SmallString<32> xmrString;
```

Comment:
nit: some duplication here with the visitSV. would be good to unify at some point.

Reply:
This is a good observation. I'm at a loss as to how to actually unify these in a clean way given that one is directly writing to a specialized pretty printing stream and the other is needing to generate a StringAttr. I'll add a TODO. 😅